docs/index.html

```diff
@@ -0,0 +1,657 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="main.css">
+    <meta charset="UTF-8">
+    <title>ambianic-functions-collection</title>
+                                                         </head>
+<body>
+<div class="container">
+    <h1>ambianic-functions-collection</h1>
+    <p class="sw-info">
+        
+        Version: <span class="sw-info-version">a7b6cde1-7435-479a-bd1a-20ab61f9ae08</span>,     <a href="" class="sw-external-doc"></a>
+    
+    </p>
+    <p><p>A postman collection documenting the Netlify cloud functions created for use within Ambianic-PWA</p>
+    </p>
+    
+    
+    <div id="sw-schemes" class="sw-default-value">
+        <span class="sw-default-value-header">Schemes:</span>
+        
+    </div>
+    
+        <h2 id="swagger--summary-no-tags">Summary</h2>
+        <table class="table table-bordered table-condensed swagger--summary">
+            <thead>
+            <tr>
+                <th>Path</th>
+                <th>Operation</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+                    <tr>
+                            <td class="swagger--summary-path" rowspan="1">
+                                <a href="#path--notification">/notification</a>
+                            </td>
+                        <td>
+                            <a href="#operation--notification-post">POST</a>
+                        </td>
+                        <td>
+                            <p>Send an event detection notification</p>
+        
+                        </td>
+                    </tr>
+                    <tr>
+                            <td class="swagger--summary-path" rowspan="1">
+                                <a href="#path--subscribe">/subscribe</a>
```

Comment:
@vickywane doc is out of sync with code. Still refers to 'POST /subscribe' instead of 'POST /subscription`
Documentation should be generated and updated automatically when the openapi spec document is updated. One source of truth to avoid this type of divergence of content.

Reply:
Recreating the ghpages docs is part of the CI process and It happens before the docs are published.